swagger_django_generator/templates/django/utils.py

```diff
@@ -57,6 +57,14 @@ def wrapper(request, *args, **kwargs):
                         request.user = user
                         return view_func(request, *args, **kwargs)
 
+                    if "HTTP_X_API_KEY" in request.META:
```

Comment:
This block is not at the right level. The two `if <header> in request.META` statements should be at the same level.